frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java

```diff
@@ -55,28 +57,57 @@ private static SchedulerBuilder createSchedulerBuilder(File yamlSpecFile) throws
         File configDir = yamlSpecFile.getParentFile();
 
         SchedulerConfig schedulerConfig = SchedulerConfig.fromEnv();
+
+        Map<String, String> env = System.getenv();
+
+        String userAuthMapping;
+
+        if (Boolean.valueOf(env.get("TASKCFG_ALL_SECURITY_KERBEROS_ENABLED"))) {
+            String frameworkHostname = EndpointUtils.toAutoIpDomain(rawServiceSpec.getName(), schedulerConfig);
```

Comment:
It might be useful to link this to `FRAMEWORK_HOST` in the variable name?

Reply:
I am not sure what you by this. Isn't the variable name `frameworkHostname` self-explaining?

Reply:
I mean that we use `frameworkHost` instead of `frameworkHostname`.